akka-stream/src/main/scala/akka/stream/javadsl/Flow.scala

```diff
@@ -1064,6 +1064,57 @@ final class Flow[-In, +Out, +Mat](delegate: scaladsl.Flow[In, Out, Mat]) extends
     new Flow(delegate.keepAlive(maxIdle, () ⇒ injectedElem.create()))
 
   /**
+   * Sends elements downstream with maximum speed of `elements/per`. In other words, this stage set the maximum rate
+   * for emitting messages. This combinator works for streams where all elements have the same cost or length.
+   *
+   * If there were no messages for sometime, throttle can emit `maximumBurst` number of elements at once, without
+   * waiting.
+   *
+   * Parameter `mode` manages behaviour when upstream is faster than throttle rate:
+   *  - [[akka.stream.ThrottleMode.ShapingMode]] makes pauses before emitting messages to meet throttle rate
+   *  - [[akka.stream.ThrottleMode.EnforcingMode]] fails with exception when upstream is faster than throttle rate. This
+   *  can be used for http servers to control SLA
```

Comment:
It is more like a quota than an SLA. But don't mention it here, HTTP quotas are more complex as most of the time there are multiple frontends which this stage does not handle (throttling is local to the stream it is applied to).